tests/tensorflow/keras/test_keras_mirrored.py

```diff
@@ -357,9 +357,9 @@ def test_save_one_worker(out_dir):
     assert len(tr.steps())
     assert len(tr.tensor_names(collection="weights"))
     assert len(tr.tensor_names(collection="weights"))
-    assert len(tr.tensor(tr.tensor_names(collection="weights")[0]).workers(0)) == 1
+    assert len(tr.tensor(tr.tensor_names(collection="weights")[0]).workers(5)) == 1
     assert len(tr.tensor_names(collection="biases"))
-    assert len(tr.tensor(tr.tensor_names(collection="biases")[0]).workers(0)) == 1
+    assert len(tr.tensor(tr.tensor_names(collection="biases")[0]).workers(5)) == 1
```

Comment:
reason for this change?

Reply:
save_config has save_steps=[5]. the asserts check step number 0 and error out.